.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,173 @@
+          ${{ inputs.name }} camunda/camunda-platform
+          -f benchmarks/setup/default/zeebe-values.yaml
+          --set zeebe.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe.image.tag=${{ inputs.name }}
+          --set zeebe-gateway.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe-gateway.image.tag=${{ inputs.name }}
+  deploy-benchmark-load:
+    name: Deploy benchmark load
+    needs:
+      - deploy-benchmark-cluster
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - uses: google-github-actions/get-gke-credentials@v0.8.2
+        with:
+          cluster_name: ${{ inputs.cluster }}
+          location: ${{ inputs.cluster-region }}
+      - name: Deploy and configure additional benchmark components
+        uses: actions/github-script@v6
+        with:
+          # language=javascript
+          script: |
+            const namespace = context.payload.inputs.name
+            const load = context.payload.inputs.load
+            const components = load.split(", ")
+            for (const component of components) {
+              if (component.includes("=")) {
+                const nameAndRate = component.split("=")
+                const name = nameAndRate.at(0)
+                const rate = nameAndRate.at(1)
+                const manifest = `benchmarks/setup/default/${name}.yaml`
+                await exec.exec('kubectl', ['delete',
+                  `--ignore-not-found=true`,
+                  `--namespace=${namespace}`,
+                  'configmap', `${name}-config`])
+                await exec.exec('kubectl', ['create',
+                  `--namespace=${namespace}`, 'configmap', `${name}-config`,
+                  `--from-literal=CONFIG_FORCE_app_starter_rate=${rate}`,
+                  `--from-literal=CONFIG_FORCE_app_brokerUrl=${namespace}-zeebe-gateway:26500`])
+                await exec.exec(`kubectl apply -n ${namespace} -f ${manifest}`)
+              } else {
+                const manifest = `benchmarks/setup/default/${component}.yaml`
+                await exec.exec('kubectl', ['delete',
+                  `--ignore-not-found=true`,
+                  `--namespace=${namespace}`,
+                  'configmap', `${component}-config`])
+                await exec.exec('kubectl', ['create',
+                  `--namespace=${namespace}`,
+                  'configmap', `${component}-config`,
+                  `--from-literal=CONFIG_FORCE_app_brokerUrl=${namespace}-zeebe-gateway:26500`])
+                await exec.exec(`kubectl apply -n ${namespace} -f ${manifest}`)
+              }
```

Comment:
:question:  I somehow feel this is far too complicated. Couldn't we just use the newBenchmark and makefile? Or is there any issue with it? :thinking:

Reply:
Tbh. I prefer this over bash scripts. I could imagine that there is a nice middle ground though: Have _one_ script in a nice high level language (not bash :sweat_smile:) that supports everything that's needed for this workflow but that can also be run locally.

Reply:
> (not bash ￼)

:laughing: 

> that supports everything that's needed for this workflow but that can also be run locally.

That would be great. I guess python :shrug: :D

Reply:
Are you okay with doing that in a follow up issue though?  I haven't touched the scripts so far so we can still use them.

Reply:
Looking at it I'm not sure why this needs a script in general? Why not just prepare the environment, then run a bunch of commands? The only logic here is splitting over `=` and iterating over the components. idk :shrug:

Reply:
> Why not just prepare the environment, then run a bunch of commands

Sure, but I wasn't very keen on figuring out the right bash incantations  for splitting a comma separated list and splitting on `=` :shrug: 
I'm happy to merge improvements though!